terraform/application/cloud_run.tf

```
@@ -0,0 +1,58 @@
+locals {
+  image_url = "${var.region}-docker.pkg.dev/${var.google_project_id}/${google_artifact_registry_repository.hapdfy.repository_id}/hapdfy@${var.image_digest}"
+}
+
+resource "google_cloud_run_service" "hapdfy" {
+  depends_on = [
+    google_project_iam_member.hapdfy
+  ]
+  name     = "hapdfy"
```

```suggestion
  name     = "{var.prefix}hapdfy"
```
As both stage and live run in the same project all named resources need to have the prefix applied